packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -301,12 +302,72 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+    regularLabel: [
+      classNames.regularLabel,
+      {
+        position: 'absolute',
+        height: '0px',
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      {
+        transform: 'translateY(6px) translateX(-6px)'
+      },
+      vertical && {
+        margin: '0 auto',
+        marginBottom: '20px',
+        transform: getRTL() ? 'translateX(6px)' : 'translateX(-6px)'
+      },
+      disabled && {
+        background: theme.palette.neutralLight,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'GrayText'
+          }
+        }
+      }
+    ],
+
+    regularTick: [
+      classNames.regularTick,
+      {
+        position: 'absolute',
+        background: '#fff',
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      disabled! && {
+        background: theme.palette.neutralLight,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'GrayText'
+          }
+        }
+      },
+      vertical
+        ? {
+            width: '16px',
+            height: '1px',
+            transform: getRTL() ? 'translateX(6px)' : 'translateX(-6px)'
```

Comment:
Same here as with the other ternary. Knowing that horizontal is default you can apply default styles in the first style object given to this style section and then change them only when vertical flag is true.

Reply:
ok just did